doc/src/manual/running-external-programs.md

```diff
@@ -59,7 +59,32 @@ julia> open(`less`, "w", STDOUT) do io
 2
 3
 ```
+## Cmd Object
```

Comment:
For consistency with the rest of this file's style, add blank lines separating, e.g., headings, doctests, paragraphs etc from one another?